Library/brew.rb

```diff
@@ -94,14 +94,8 @@ def require?(path)
     exit Homebrew.failed? ? 1 : 0
   else
     require "tap"
-    possible_tap = case cmd
-    when "brewdle", "brewdler", "bundle", "bundler"
-      Tap.fetch("Homebrew", "bundle")
-    when "cask"
-      Tap.fetch("caskroom", "cask")
-    when "services"
-      Tap.fetch("Homebrew", "services")
-    end
+    possible_tap = OFFICIAL_CMD_TAPS.find { |_, repo| repo == cmd }
```

Comment:
Why don't we use the dictionary approach in the above? i.e.
```ruby
OFFICIAL_CMD_TAPS = {
  "homebrew/bundle" => %w[bundle, ...]
  ...
 }.freeze
```

I however opposite to only check cmd with tap name, which it's a bad assumption for future commands or taps.